app/controllers/meetings_controller.rb

```diff
@@ -57,7 +57,7 @@ def delete_comment
       Notification.where(uniqueid: public_uniqueid).destroy_all
     end
 
-    render nothing: true
+    head :ok
```

Comment:
Could you explain why you changed this and if this behaviour is equivalent to what was previously there?

Reply:
When I wrote the tests for delete_comment I got this warning.
```
 .DEPRECATION WARNING: `:nothing` option is deprecated and will be removed in Rails 5.1. Use `head` method to respond with empty response body. (called from delete_comment at ...ifme/app/controllers/meetings_controller.rb:60)
```

Searching for solutions I found this answer explaining to use 'head :ok'. 
https://stackoverflow.com/questions/34688726/the-nothing-option-is-deprecated-and-will-be-removed-in-rails-5-1

Reply:
Oooh awesome catch/find!